src/main/java/com/google/api/generator/gapic/composer/samplecode/ServiceClientSampleCodeComposer.java

```diff
@@ -253,7 +254,11 @@ public static String composeRpcDefaultMethodHeaderSampleCode(
             Variable.builder().setName("request").setType(method.inputType()).build());
     List<VariableExpr> rpcMethodArgVarExprs = Arrays.asList(requestVarExpr);
     Message requestMessage = messageTypes.get(method.inputType().reference().simpleName());
-    Preconditions.checkNotNull(requestMessage);
+    Preconditions.checkNotNull(
+        requestMessage,
+        String.format(
+            "The request `%s` is not defined as message type.",
```

Comment:
Could we please make the error message more accurate? E.g. "Could not find the message type %s".

Reply:
Will update the error message.